### PR TITLE
Update tool.md

### PR DESCRIPTION
**Pull request by SpBills**

### CODE REVIEW

File: Dev-Tools/aws/tool.md [hunk 1]
```diff
@@ -4,16 +4,70 @@
...
-If you need to install anything add the steps in here. If is just a website, add the link to the description, and remove this section.
+There is no installation needed for AWS. AWS is a web service. Depending on your exec, they will either issue everyone on the team an account, or only give out accounts to specific people. Please contact your exec to understand if you have an account.
+
+https://us-east-2.signin.aws.amazon.com/
+
+Our account ID: 177987852765
 
 ## How to use
 
 This will be the most important section. Describe (with pictures) how to set up projects, navigate the web app, and troubleshoot. Anything that can be useful should exist here.
```
Comment (SebastianNeiswanger) [inline Dev-Tools/aws/tool.md]: Can you remove this line? It was meant as description of what to put in this section.

**Review by SebastianNeiswanger — CHANGES_REQUESTED** [review]
Just this one comment everything else looks great here.

File: Dev-Tools/aws/tool.md (hunk 1 above)
Comment (SebastianNeiswanger) [reply Dev-Tools/aws/tool.md]: Let me specify line 21

Comment (SpBills) [reply Dev-Tools/aws/tool.md]: My b

**Comment by SpBills** [comment]
@SebastianNeiswanger ready

**Review by SebastianNeiswanger — APPROVED** [review]
lgtm

**Pull request merged**